dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/icon/IconTest.java

```diff
@@ -88,12 +88,34 @@ protected void initTest() throws IOException {
   void shouldGetAllIconsWhenRequested() {
     Map<String, DefaultIcon> defaultIconMap = getAllDefaultIcons();
 
+    IconOperationParams operationParams = new IconOperationParams();
+    operationParams.setIconTypeFilter(IconTypeFilter.ALL);
+    operationParams.setKeywords(List.of());
+
     assertEquals(
         defaultIconMap.size() + 1,
-        iconService.getIcons().size(),
+        iconService.getIcons(operationParams).size(),
```

Comment:
Tests to me are easier to read if they follow the same structure. I would like to see with a glimpse whats setup, whats being tested/called, whats the expected outcome. Having three groups for each achieves that for example.

I would at least extract the `iconService.getIcons(operationParams)` which is whats tested AFAIK

Reply:
fixed